music21/stream/base.py

```diff
@@ -4932,6 +4932,20 @@ def _transposeByInstrument(self,
             # print(k, i.transposition)
         return returnObj
 
+    def _shouldBeAtSoundingPitch(self, contextStream):
```

Comment:
I get this and why it needs to be here, but maybe a little more docs about why internal Streams might have atSoundingPitch = 'unknown' (Measures, etc.) while the Score has a transposition set